client/app/components/groupDetail/groupDetail.controller.js

```diff
@@ -1,6 +1,13 @@
 class GroupDetailController {
-  constructor(Group, CurrentGroup, $state, $stateParams) {
+  constructor(Group, CurrentGroup, $mdMedia, $scope, $state, $stateParams) {
     "ngInject";
+    $scope.screenIsSmall = !$mdMedia("gt-sm");
```

Comment:
Code style thing: we could outsource this to a service, if we want to have one application-wide definition of what _screen is small_ means. But it's also ok like this.